README.md

```diff
@@ -76,12 +76,34 @@ be found in [Loadero wiki](https://wiki.loadero.com/docs/testui-python/custom-co
-| `ignore_alert`               | No differences                                                                         |
-| `set_file`                   | Any local file can be used, Loadero constant can be used if the same file name is used |
-| `set_request_header`         | No request header will be set                                                          |
-| `set_user_agent`             | User agent won't be changed                                                            |
-| `time_execution`             | Execution time will be logged, but not saved                                           |
-| `update_network`             | Network settings will not be updated                                                   |
-| `wait_for_download_finished` | Function will finish instantly and not wait for download to be finished                |
+| Command                      | Differences                                                                                                                                   |
+| ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ignore_alert`               | No differences                                                                                                                                |
+| `receive_email`              | By default an empty array will be returned, but `emails.json` file can be saved in the working directory which will contain all of the emails |
+| `gen_email`                  | Full email or only first part can be provided, if only the first part is provided than `@mailinator` is appended                              |
+| `set_file`                   | Any local file can be used, Loadero constant can be used if the same file name is used                                                        |
+| `set_request_header`         | No request header will be set                                                                                                                 |
+| `set_user_agent`             | User agent won't be changed                                                                                                                   |
+| `time_execution`             | Execution time will be logged, but not saved                                                                                                  |
+| `update_network`             | Network settings will not be updated                                                                                                          |
+| `wait_for_download_finished` | Function will finish instantly and not wait for download to be finished                                                                       |
+
+The `emails.json` file should be located in the directory from which you will be running the tests and should have the following structure:
```

Comment:
doesn't seem that this line is wrapped at 80 chars; might add a note (similar as for NW) that file can be created mid-test to simulate delay in receiving the message